qiskit/qpy/binary_io/value.py

```diff
@@ -1003,6 +974,7 @@ def dumps_value(
     Raises:
         QpyError: Serializer for given format is not ready.
     """
+    # patch to ensure Parameter objects are treated as such
```

Comment:
Anything needed here?

Reply:
Removed in: https://github.com/Qiskit/qiskit/pull/14799/commits/0c59d09fc85363b2e244291bbd3692527b1eeb38